src/plugins/PositionPlugin.cpp

```diff
@@ -121,7 +121,8 @@ void PositionPlugin::sendOurPosition(NodeNum dest, bool wantReplies)
     p->priority = MeshPacket_Priority_BACKGROUND;
     prevPacketId = p->id;
 
-    service.sendToMesh(p);
+    //send to the mesh, and the phone
```

Comment:
@joshpirihi, I'm trying to test the PR checks for 1.2-legacy. Would you mind adding a space here and pushing it so that it kicks off a build? Doesn't seem like it triggers on existing PRs

Reply:
Like that?

Reply:
Perfect!